web/templates/job/index.html.eex

```diff
@@ -1,9 +1,20 @@
-<ul>
-  <%= for job <- @jobs  do %>
-    <li>
-      <%= job.title %>
-      <%= job.city %>
-      <%= job.company %>
-    </li>
-  <% end %>
-</ul>
+<section class="section">
+  <header class="section-description">
+    <h4>Job Board</h4>
+    <p>Find the best design jobs local to Raleigh, NC, include those for UI,
+    UX, graphic, and illustration&amp;designers.<p>
+    <p>Jobs are free to post, and will stay online for 30&amp;days.</p>
```

Comment:
What's up with the `&amp;`s not showing up in the template? Is that a Phoenix thing?